### ISSUE TITLE
Show App Slug in Overview

### ISSUE THREAD

**Issue by fchamieh** [Microsoft/appcenter#239]
Showing the "App Slug" in the application overview is useful to connect to other services such as AzureDevops. 
For example the App Slug is **required** in Azure Pipelines Task: **AppCenterDistribute**.

**Comment by lumaxis** [comment]
Yes! I've personally been a big advocate of this internally before so I'd love if we can find time to actually build this 🙂
